build.groovy

```diff
@@ -711,6 +711,9 @@ def createBuildList() {
 }
 
 def finalizeBuildProcess(Map args) {
+     finalizeBuildProcess(args, 0)
```

Comment:
I would suggest we follow the same strategy and access the attributes in the map instead of overriding the `finalizeBuildProcess` method